includes/class-wc-stripe-helper.php

```diff
@@ -373,7 +375,20 @@ public static function get_webhook_url() {
 	public static function get_order_by_source_id( $source_id ) {
 		global $wpdb;
 
-		$order_id = $wpdb->get_var( $wpdb->prepare( "SELECT DISTINCT ID FROM $wpdb->posts as posts LEFT JOIN $wpdb->postmeta as meta ON posts.ID = meta.post_id WHERE meta.meta_value = %s AND meta.meta_key = %s", $source_id, '_stripe_source_id' ) );
+		if ( OrderUtil::custom_orders_table_usage_is_enabled() ) {
```

Comment:
`OrderUtil` it's been recently introduced ([~2 months ago](https://github.com/woocommerce/woocommerce/pull/33638)) so for older WC versions this will throw an error. I'll add a validation

Reply:
done b79149003bb0318dead088f6b8dacfd6bcba6fbf